### PR TITLE
Section added to Remdesivir

### PR DESCRIPTION
**Pull request by RLordan**
The recent NEMJ paper has been added. Feel free to edit, review etc. Thanks all.

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
 The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
 Remdesivir has been used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
 All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
+Indeed, the New England Medical Journal published a Gilead Sciences sponsored trial that administered 200mg of Remdesivir intravenously on day 1 followed by a further 100mg/day for 9 days on compassionate-use basis to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].
```
Comment (cgreene) [inline content/20.treatments.md:118]:
```suggestion
A recent publication, led by Gilead Sciences, the maker of Remdesivir, reported outcomes for compassionate-use of the drug administered 200mg of intravenously on day 1 followed by a further 100mg/day for 9 days to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
...
+Following treatment, data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients.
```
Comment (cgreene) [inline content/20.treatments.md:119]:
```suggestion
```

File: content/20.treatments.md [hunk 3]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
...
+While that result might seem promising, there were significant issues with the study design that undermine the seemingly positive outcome.
+These included the fact that the trial did not include a randomized control group, some patients only required low doses of oxygen, others required ventilation, patients analyzed had mixed demographics as it was a multisite trial, there was a short follow-up period of investigation, some patients worsened, some patients died, and lastly 8 patients were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
+As a result of these confounding issues, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardess of treatment.
```
Comment (cgreene) [inline content/20.treatments.md:122]:
```suggestion
As a result of these issues, though data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardless of treatment.
```

File: content/20.treatments.md [hunk 4]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
 The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
 Remdesivir has been used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
 All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
+Indeed, the New England Medical Journal published a Gilead Sciences sponsored trial that administered 200mg of Remdesivir intravenously on day 1 followed by a further 100mg/day for 9 days on compassionate-use basis to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].
+Following treatment, data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients.
+While that result might seem promising, there were significant issues with the study design that undermine the seemingly positive outcome.
```
Comment (cgreene) [inline content/20.treatments.md:120]:
```suggestion
There were significant issues with the study design or lack thereof.
```

File: content/20.treatments.md [hunk 5]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
...
+These included the fact that the trial did not include a randomized control group, some patients only required low doses of oxygen, others required ventilation, patients analyzed had mixed demographics as it was a multisite trial, there was a short follow-up period of investigation, some patients worsened, some patients died, and lastly 8 patients were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
```
Comment (cgreene) [inline content/20.treatments.md:121]:
```suggestion
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
Patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
```

File: content/20.treatments.md [hunk 6]
```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
...
+As a result of these confounding issues, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardess of treatment.
+Therefore, no conclusive deductions can be made at this time until  randomised controlled trials are completed.
```
Comment (cgreene) [inline content/20.treatments.md:123]:
```suggestion
Therefore, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```

File: content/20.treatments.md (hunk 3 above)
Comment (rando2) [reply content/20.treatments.md:122]:
```suggestion
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it cannot be determined whether treatment with Remdesivir had an effect or whether these patients would have recovered regardless of treatment.
```
Rephrasing of @cgreene's suggestion

File: content/20.treatments.md (hunk 6 above)
Comment (rando2) [reply content/20.treatments.md:123]: If you take my suggestion on the previous sentence, here I would suggest using Casey's sentence but swapping "Therefore" with "As a result"

**Comment by RLordan** [comment]
> Thanks for adding this @RLordan! I've seen this study circulating a bunch and I think you and @cgreene present the results well. I just had a few suggestions for flow (checked against the abstract of the paper, I did not read the whole paper, but it seems like Casey and @hufengling have read it in depth).

Thanks @cgreene for the edits, much improved.

**Comment by RLordan** [comment]
> Thanks for adding this @RLordan! I've seen this study circulating a bunch and I think you and @cgreene present the results well. I just had a few suggestions for flow (checked against the abstract of the paper, I did not read the whole paper, but it seems like Casey and @hufengling have read it in depth).

Thanks @rando2 I am going back to change the few words that appeared repetitive, you have great patience!

**Comment by RLordan** [comment]
Hey @rando2 there was one change of words where you suggested the change of "Therefore" to "As a result"... For some reason I can't figure out how to edit that. Can you make that suggestion again and I can commit. Sorry!

File: content/20.treatments.md [hunk 7]
```diff
@@ -115,6 +115,17 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
...
+There were significant issues with the study design or lack thereof.
+There was no randomized control group.
+The inclusion criteria were variable: some patients only required low doses of oxygen, others required ventilation.
+The study included many sites, potentially with variable inclusion criteria and treatment protocols.
+Patients analyzed had mixed demographics.
+There was a short follow-up period of investigation.
+Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
+Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it cannot be determined whether treatment with Remdesivir had an effect or whether these patients would have recovered regardless of treatment.
+Therefore, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```
Comment (RLordan) [inline content/20.treatments.md:128]:
```suggestion
As a result, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```

**Comment by cgreene** [comment]
This looks good to me! If the build passes I will approve.

**Pull request merged**